staging/src/k8s.io/code-generator/_examples/apiserver/apis/example/v1/zz_generated.conversions.go

```diff
@@ -1,113 +0,0 @@
-// +build !ignore_autogenerated
-
-/*
-Copyright 2017 The Kubernetes Authors.
```

Comment:
why is this gone?

Reply:
filename is conversion(s), also has same file zz_generated.conversion.go in the dir, don't know why..

Reply:
this one is concerning, shouldn't happen

Reply:
oh, did the name change? If we still have zz_generated.conversion.go here, it's probably ok.

Reply:
Can have a look in the dir https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/code-generator/_examples/apiserver/apis/example/v1   :)